scripts/onset_zscore_connectivity_matrix.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+"""
+Compute z-score matrices from base matrices using mean and std connectivity matrices.
+"""
+
+import argparse
+import logging
+import numpy as np
+from typing import List
+
+from onsetpy.io.matrix import save_matrix, load_matrix
+from onsetpy.io.utils import (
+    add_verbose_arg,
+    add_overwrite_arg,
+    assert_inputs_exist,
+    assert_matrices_compatible,
+    assert_outputs_exist,
+)
+
+
+def calculate_z_scores(
+    mean_matrix: np.ndarray, std_matrix: np.ndarray, base_matrices: List[np.ndarray]
+) -> List[np.ndarray]:
+    """Compute z-score matrices for each base matrix.
+
+    Args:
+        mean_matrix (np.ndarray): Mean connectivity matrix.
+        std_matrix (np.ndarray): Standard deviation connectivity matrix.
+        base_matrices (List[np.ndarray]): List of base connectivity matrices.
+
+    Returns:
+        List[np.ndarray]: List of z-score matrices.
+    """
+    z_score_matrices = []
+    for base_matrix in base_matrices:
+        z_score_matrix = (base_matrix - mean_matrix) / std_matrix
+        z_score_matrices.append(z_score_matrix)
+    return z_score_matrices
+
+
+def _build_arg_parser():
+    """Build argparser.
+
+    Returns:
+        parser (ArgumentParser): Parser built.
+    """
+    parser = argparse.ArgumentParser(
+        description=__doc__, formatter_class=argparse.RawTextHelpFormatter
+    )
+    parser.add_argument(
+        "--mean", required=True, help="Path to the mean connectivity matrix in .npy format"

```

Comment:
Why do you put mean and std as a required option ? You can just declare them as a mandatory argument (as base_matrices)